Cargo.toml

```diff
@@ -27,6 +27,7 @@ strip = true
 anyhow = { version = "1.0", features = ["backtrace"] }
 ar = "0.9"
 async-trait = "0.1"
+backon = "0.4.1"
```

Comment:
This line is required for the project to build, since https://github.com/apache/incubator-opendal/commit/2c2dfc8f994d14bcbb16d44ef3603081fe9e0745 is missing in opendal 0.39.